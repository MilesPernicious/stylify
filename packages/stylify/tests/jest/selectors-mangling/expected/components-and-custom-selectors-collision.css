.docs__section-link{
	font-weight: bold
}
.a{
	color: blue
}
