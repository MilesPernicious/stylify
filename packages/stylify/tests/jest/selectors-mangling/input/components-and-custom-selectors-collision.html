<div class="docs__section-link link"></div>
