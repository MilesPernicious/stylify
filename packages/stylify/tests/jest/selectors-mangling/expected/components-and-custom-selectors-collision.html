<div class="docs__section-link a"></div>
